CONTRIBUTING.md

```diff
@@ -1,77 +1,109 @@
+- [Merging pull requests](#merging-pull-requests)
 
-## Submitting Issues
-
-You can submit issues with respect to the vagrant-service-manager plugin [here](https://github.com/projectatomic/vagrant-service-manager/issues/new).Make sure you include all the relevant details pertaining the issue.
-
-Before submitting new issues, it is suggested to check [all existing issues](https://github.com/projectatomic/vagrant-service-manager/issues) in order to avoid duplication.The vagrant-service-manager plugin works closely with the [ADB](https://github.com/projectatomic/adb-atomic-developer-bundle/issues) and the [adb-utils](https://github.com/projectatomic/adb-utils/issues) RPM. You may wish to review the issues in both the repositories as well.
+<!-- /MarkdownTOC -->
 
+The following is a set of guidelines for contributing to the
+vagrant-service-manager plugin.
 
-## Submitting Pull Requests
+These are guidelines, please use your best judgment and feel free to propose
+changes to this document.
 
-* All changes will be made by pull request (PR), even from core
-  committers/maintainers.
+<a name="submitting-issues"></a>
+## Submitting issues
 
-* All changes must include appropriate documentation updates.
+You can submit issues with respect to the vagrant-service-manager plugin [here](https://github.com/projectatomic/vagrant-service-manager/issues/new).
+Make sure you include all the relevant details pertaining the issue.
 
-* All changes must include an entry in the [Changelog document](https://github.com/projectatomic/vagrant-service-manager/blob/master/CHANGELOG.md) in the *Unreleased* section describing the change. Your new entry should be the last entry in the *Unreleased* section and should include your GitHub userid.
+Before submitting a new issue, it is suggested to check the [existing issues](https://github.com/projectatomic/vagrant-service-manager/issues)
+in order to avoid duplication. The vagrant-service-manager plugin works closely
+with the [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-developer-bundle/issues)
+and the [adb-utils](https://github.com/projectatomic/adb-utils/issues) RPM.
+You may wish to review the issues in both these repositories as well.
 
-* All changes need at least 2 ACKs from maintainers before they will be merged. If
-  the author of the PR is a maintainer, their submission is considered
-  to be the first ACK.  Therefore, PRs from maintainers only need one
-  additional ACK.
+<a name="submitting-pull-requests"></a>
+## Submitting pull requests
 
-  By "2 ACKs" we mean that two maintainers must acknowledge that the change
-  is a good one. The 2<sup>nd</sup> person to ACK the PR should merge the PR with
-  a comment including their agreement.
+<a name="get-started"></a>
+### Get Started
 
+If you are just getting started with Git and GitHub there are a few prerequisite
+steps.
 
-## How to Develop/Test
+* Make sure you have a [GitHub account](https://github.com/signup/free).
```

Comment:
Instead of this instructions, I like the approach of giving copy-paste commands like
1. Fork the repo
2. Git clone as:
```
$ git clone https://github.com/<YOUR_USERNAME>/vagrant-service-manager.git
```
and others too